pkg/util/users/users.go

```diff
@@ -120,6 +118,13 @@ func (strategy DefaultUserSyncStrategy) ExecuteSyncRequests(requests []spec.PgSy
 			if err := strategy.alterPgUser(request.User, db); err != nil {
 				reqretries = append(reqretries, request)
 				errors = append(errors, fmt.Sprintf("could not alter user %q: %v", request.User.Name, err))
+				// check if additional owners are misconfigured as members to a database owner
```

Comment:
that line well-deserved a comment with the PR number so that a reader can find more context on why the misconfiguration was ever possible